provision/openshift/Taskfile.yaml

```diff
@@ -198,19 +195,16 @@ tasks:
         if [ "{{.KC_ISPN_NAMESPACE}}" != "" ]; then
         echo "true" > .task/remote-store-enabled
         echo "infinispan.{{.KC_ISPN_NAMESPACE}}.svc" > .task/remote-store-host
-        oc -n {{.KC_ISPN_NAMESPACE}} get secrets infinispan-generated-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].username > .task/remote-store-username
-        oc -n {{.KC_ISPN_NAMESPACE}} get secrets infinispan-generated-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].password > .task/remote-store-password
+        oc -n {{.KC_ISPN_NAMESPACE}} get secrets connect-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].password > .task/remote-store-password
```

Comment:
Should I keep fetching the password from the secret or convert this toa variable?

Reply:
is this a secret generated by the operator ? @pruivo how much of an effort would it be to fetch it from the IAM manager ? `--secret-id "$KEYCLOAK_MASTER_PASSWORD_SECRET_NAME"`